### PR TITLE
add user billing stuff

### PR DESCRIPTION
**Pull request by GusCaplan**

### CODE REVIEW

File: src/structures/Payment.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    this.currency = data.currency;
+
+    /**
+     * Amount charged, if any
+     * @type {number}
+     */
+    this.amount = data.amount;
+
+    /**
+     * Amound refunded, if any
```
Comment (hydrabolt) [inline src/structures/Payment.js]: Typo

File: src/structures/Payment.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+     * Time the payment was created
+     * @type {Date}
+     */
+    this.createdAt = new Date(data.created_at);
+
+    /**
+     * Time the payment was created
+     * @type {number}
+     */
+    this.createdTimestamp = this.createdAt.getTime();
```
Comment (hydrabolt) [inline src/structures/Payment.js]: not sure whether this should be made instead of created, check back later

Comment (GusCaplan) [inline src/structures/Payment.js]: this should 100% be created

**Comment by Programmix** [comment]
> something was added to API
> I, Gus, shall add it to discord.js, regardless of how useless it is.

**Comment by Gawdl3y** [comment]
I don't think this should be in the lib at all, actually. Not only is there literally no use-case that I'm aware of, but it adds data that would be open to accidental exposure, that nobody wants to give out.

**Comment by PgBiel** [comment]
^

**Comment by GusCaplan** [comment]
"accidental exposure" 🤔  lets break this down

accidental: you would have to 'accidentally` run this, grab the output of the promise, and then get that to some forum of exposure, which brings us to the exposure part.

exposure: there is nothing returned in these endpoints that is unsafe for other people to see, with the most revealing part being the last4 of your cc

**Comment by Programmix** [comment]
Can you provide a use case for these changes?

**Comment by Tyler0130** [comment]
a) On that matter of exposure, I feel that's a poor excuse to not add it to the library. Not putting it in the library doesn't stop people from using the endpoints.

b) On the use case front, while you may not have a use case, that doesn't mean someone else wouldn't find this useful. They may not even know they would find it useful until they see it. The mere fact that someone thought to put it in the library should tell you there's a possible use case.

That's just my two cents.

**Comment by GusCaplan** [comment]
closing because tyler agreed with me

**Pull request closed without merging**